server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/scans.js

```diff
@@ -18,6 +18,8 @@
  */
 "use strict";
 
+var scansList;
```

Comment:
Can this be local to the function?
As in `var scansList = $('#scansList'). ...`

Reply:
`scansList` is also used in the `refreshScansTable` function. Wouldn't it complain from being out of scope?

Reply:
Are they referencing the same value, or just reusing a variable name? If they are just reusing a variable name, both can be locally scoped. If the idea is to declare a global in one method, and reading the variable in the other, then this change is probably what we want. I'm not sure which is the case without looking at the other method.

Reply:
`if (scansList)
scansList.ajax.reload(null, false);`

In the other method, an if statement checks for the existence of scansList, so I think it would be referencing the same scansList var created in the previous method.